GameServerScripts/GameServerScripts.csproj

```diff
@@ -149,6 +149,9 @@
     <Compile Include="globals.cs">
       <SubType>Code</SubType>
     </Compile>
+    <Compile Include="quests\Albion\GreetingsArmsman.cs" />
+    <Compile Include="quests\Albion\GreetingsPaladin.cs" />
+    <Compile Include="quests\Hibernia\BonesToBlades.cs" />

```

Comment:
What are the changes to this file fore?

Reply:
Files were not in project but should have been.